### PR TITLE
Request model BE

### PR DESCRIPTION
**Pull request by ruslan-akhm**
- request model as per #32

### CODE REVIEW

File: server/models/requestModel.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    type: Schema.Types.ObjectId,
+    ref: "Profile",
+    required: true,
+  },
+  sitterId: {
+    type: String,
+    required: true,
+  },
+  start: {
+    type: Number,
```
Comment (dproc96) [inline server/models/requestModel.js]: This and end should probably be dates

File: server/models/requestModel.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema({
+  userId: {
+    type: Schema.Types.ObjectId,
+    ref: "Profile",
+    required: true,
+  },
+  sitterId: {
+    type: String,
```
Comment (dproc96) [inline server/models/requestModel.js]: This should also be a ref to profile

**Review by dproc96 — CHANGES_REQUESTED** [review]
A couple of changes to the model, also in general if you're working on multiple unrelated PRs, both should be based off of dev so that they are distinct and it's easy to see what the changes are. This is capturing changes from #51

File: server/models/requestModel.js (hunk 2 above)
Comment (aeciorc) [reply server/models/requestModel.js]: Ler's also add indices to both sitterid and userId. Maybe rename the latter to ownerId to make it more clear

**Comment by ruslan-akhm** [comment]
Updated model

**Review by dproc96 — APPROVED** [review]

**Pull request merged**